cellprofiler/__main__.py

```diff
@@ -575,18 +587,23 @@ def get_batch_commands(filename):
                 print "CellProfiler -c -r -p %s -f %d -l %d" % (filename, prev + 1, off)
 
                 prev = off
+    else:
+        metadata_tags = m.get_grouping_tags()
 
-            return
-
-    metadata_tags = m.get_grouping_tags()
-
-    groupings = m.get_groupings(metadata_tags)
-
-    for grouping in groupings:
-        group_string = ",".join(["%s=%s" % (k, v) for k, v in grouping[0].iteritems()])
+        if len(metadata_tags) == 1 and metadata_tags[0] == 'ImageNumber':
+            for i in range(0, len(image_numbers), n_per_job):
+                first = image_numbers[i]
+                last = image_numbers[min(i+n_per_job-1, len(image_numbers)-1)]
+                print "CellProfiler -c -r -p %s -f %d -l %d" % (filename, first, last)
+        else:
+            # LoadData w/ images grouped by metadata tags
```

Comment:
I learned that this code block is executed when a user creates a pipeline using LoadData and elects to group images by metadata tags.

/cc @VolkerH